### PR TITLE
Add Sapling details to release notes

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -4,6 +4,47 @@ release-notes at release time)
...
+-----------------------
+
+The consensus code preparations for the Sapling network upgrade, as described
+in [ZIP 243](https://github.com/zcash/zips/blob/master/zip-0243.rst) and the
+[Sapling spec](https://github.com/zcash/zips/blob/master/protocol/sapling.pdf)
+are finished and included in this release. Sapling support in the wallet and
+RPC is ongoing, and is expected to land in master over the next few weeks.
+
+The [Sapling MPC](https://blog.z.cash/announcing-the-sapling-mpc/) is currently
+working on producing the final Sapling parameters. In the meantime, Sapling will
```
Comment (arielgabizon) [inline doc/release-notes.md]: Should the height be decided before this is merged?

File: doc/release-notes.md [hunk 2]
```diff
@@ -4,6 +4,47 @@ release-notes at release time)
...
+activate on testnet with dummy Sapling parameters at height XXXXXX. This
+activation will be temporary, and the testnet will be rolled back by version
+2.0.0 so that both mainnet and testnet will be using the same parameters.
+Users who want to continue testing Overwinter should continue to run version
+1.1.0 on testnet, and then upgrade to 2.0.0 (which will be released after
+Overwinter activates).
+
+Sapling can also be activated at a specific height in regtest mode by
+setting the config options `-nuparams=5ba81b19:HEIGHT -nuparams=76b809bb:HEIGHT`.
```
[the remark this answers is not in the document]
Comment (arielgabizon) [reply doc/release-notes.md]: image

[the remark this answers is not in the document]
Comment (arielgabizon) [reply doc/release-notes.md]: Getting errors when trying this on regtest

File: doc/release-notes.md [hunk 3]
```diff
@@ -4,6 +4,47 @@ release-notes at release time)
...
+
+The consensus code preparations for the Sapling network upgrade, as described
+in [ZIP 243](https://github.com/zcash/zips/blob/master/zip-0243.rst) and the
+[Sapling spec](https://github.com/zcash/zips/blob/master/protocol/sapling.pdf)
+are finished and included in this release. Sapling support in the wallet and
+RPC is ongoing, and is expected to land in master over the next few weeks.
+
+The [Sapling MPC](https://blog.z.cash/announcing-the-sapling-mpc/) is currently
+working on producing the final Sapling parameters. In the meantime, Sapling will
+activate on testnet with dummy Sapling parameters at height XXXXXX. This
```
[the remark this answers is not in the document]
Comment (str4d) [reply doc/release-notes.md]: The height will be set in the release PR; I will update this placeholder at that time.

File: doc/release-notes.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (str4d) [reply doc/release-notes.md]: > Do I write the same `HEIGHT` in both places?

Yeah, that was the intent (so Overwinter and Sapling update at the same time, which does technically work).

> Getting errors when trying this on regtest

Are you running the correct binary version? I compiled current master, and it works fine for me.

[the remark this answers is not in the document]
Comment (arielgabizon) [reply doc/release-notes.md]: Oopps I compiled the right version but ran the default one on my system.  Yeah it works.

[the remark this answers is not in the document]
Comment (arielgabizon) [reply doc/release-notes.md]: ..but what do the two parameters mean then?  Cause it agrees to run also when giving different values.
image

[the remark this answers is not in the document]
Comment (arielgabizon) [reply doc/release-notes.md]: But if the first parameter is overwinter activation height, why does it agree to run when the first parameter is larger:
image

**Comment by str4d** [comment]
@zkbot r+

File: doc/release-notes.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (arielgabizon) [reply doc/release-notes.md]: Then I would add a parenthetical remark "(The first parameter is the overwinter activation height and should be set smaller or equal than the second)"

**Pull request merged**